### DISCUSSION TITLE
Is it possible to skip fulltext catalogs?

### DISCUSSION THREAD

**Discussion by Jogai** [olahallengren/sql-server-maintenance-solution#271] [Questions & Answers]
For a while I used backup from the azure UI to create a bacpac, but since the addition of fulltext search I cant restore those on our dev machines (with localdb). Can this script do a partial backup leaving out the fulltext features?

**Comment by olahallengren** [comment]
No, there is no option for that.

**Comment by wischi-chr** [comment]
@Jogai a few minutes ago I ran into the same problem (trying to import an azure bacpac backup locally). I found this issue while looking for a solution.

You can find my solution here:
https://dba.stackexchange.com/a/260714/33001

And here are the steps (copy/paste) just in case:
1. Rename your backup to zip
2. extract model.xml and Origin.xml
3. open model.xml with a text editor and look for "fulltext"
4. Delete all fulltext elements in the xml
5. set fulltext to false in the header
6. Calculate the SHA256 hash of the edited xml (I used hashtab for that)
7. Fix the hash/checksum in origin.xml
8. Add the updated files back to the zip
9. Rename back to bacpac and import
10. "Worked on my machine" - I hope it helps
